README.md

```diff
@@ -177,7 +177,8 @@ details of the core Pulumi CLI and [programming model concepts](https://www.pulu
 | <img src="https://www.pulumi.com/logos/tech/logo-js.png" height=38 />     | [JavaScript](./sdk/nodejs) | Stable  | Node.js 8+  |
 | <img src="https://www.pulumi.com/logos/tech/logo-ts.png" height=38 />     | [TypeScript](./sdk/nodejs) | Stable  | Node.js 8+  |
 | <img src="https://www.pulumi.com/logos/tech/logo-python.png" height=38 /> | [Python](./sdk/python)     | Stable  | Python 3.6+ |
-| <img src="https://www.pulumi.com/logos/tech/logo-golang.png" height=38 /> | [Go](./sdk/go)             | Preview | Go 1.x      |
+| <img src="https://www.pulumi.com/logos/tech/logo-golang.png" height=38 /> | [Go](./sdk/go)             | Stable  | Go 1.13.x   |
+| <img src="https://www.pulumi.com/logos/tech/dotnet.png" height=38 />      | [DotNet](./sdk/dotnet)     | Stable  | DotNet 3.1  |
```

Comment:
```suggestion
| <img src="https://www.pulumi.com/logos/tech/dotnet.png" height=38 />      | [.NET](./sdk/dotnet)     | Stable  | .NET Core 3.1  |
```

Reply:
@mikhailshilkov and I just spoke about this so thank you as well!